### DISCUSSION TITLE
IDCharger topics

### DISCUSSION THREAD

**Discussion by Maergsche** [tillsteinbach/WeConnect-mqtt#89] [General]
Hello, 
is it possible to get the current values of the IDCharger of the current charging process?

(The homecharging records of the current session.)

**Comment by tillsteinbach** [comment]
Hey, it is on the roadmap. You can track the progress here: https://github.com/tillsteinbach/WeConnect-python/issues/32

**Reply by Maergsche** [reply]
Perfectly, in case I can assist with testing or debugging, let me know.

(Mein nutzen für die Werte ist  zu erkennen, ob ich an der Heimischen Ladestation bin oder an einer anderen um das dann in Kombination mit EVCC zu nutzen. Also für mich wäre der aktive Ladevorgang primär interessant. )
